src/Parser/Mage/Frost/CONFIG.js

```diff
@@ -16,8 +16,14 @@ export default {
   changelog: CHANGELOG,
   description: (
     <div>
-      Hello Everyone! First things first I am by no means a theorycrafter or expert in all things mages, but my mage has been my main ever since Burning Crusade and I have a pretty good idea of how things work. <br /> <br />
-	  Additionally I am not a pro when it comes to programming these modules and analysis. So if something is missing that you think should be added, you run into an issue with something i made, or if you make a module that you think should be added, please send it to me! <br /> <br />
+      Hello Everyone! We are always looking to improve the Frost Mage Analyzers and Modules; so if you find any issues or if there is something missing that you would like to see added, please open an Issue on GitHub or send a message to us on Discord (Sharrq#7530 or Sref#3865) <br /> <br />
+	    Additionally, if you need further assistance in improving your gameplay as a Frost Mage, you can refer to the following resources:<br /> <br />
+      Discord <br />
+      https://discord.gg/0gLMHikX2aZ23VdA <br /> <br />
+      Altered Time (Mage Forums/Guides) <br />
+      https://www.altered-time.com/forum/ <br /> <br />
+      Icy Veins (Class Guides) <br />
+      https://www.icy-veins.com/wow/frost-mage-pve-dps-guide <br />
```

Comment:
All of these links can be made into actual links, and you may as well make them links with the given text showing outwardly. For example for Icy Veins it would be `<a href="https://www.icy-veins.com/wow/frost-mage-pve-dps-guide" target="_blank">Icy Veins (Class Guides)</a>`. Do the same with the other links.